dasharo-security/tpm2-commands.robot

```diff
@@ -236,6 +236,42 @@ TPMCMD011.001 Performing HMAC operation on the file (Ubuntu)
     Should Contain    ${out1}    hmac.out
     Should Not Contain    ${out2}    hmac.out
 
+TPMCMD012.001 Change EPS (Ubuntu)
+    [Documentation]    Try to trigger the TPM_RC_INTEGRITY error
+    Log To Console    \nTPMCMD0012.001 Change EPS - run
+    Execute Linux Tpm2 Tools Command    tpm2_createprimary -c primary_key.ctx    60
+    Execute Linux Tpm2 Tools Command    tpm2_create -u key.pub -r key.priv -C primary_key.ctx
+    Flush TPM Contexts
+    Execute Linux Tpm2 Tools Command    tpm2_load -C primary_key.ctx -u key.pub -r key.priv -c key.ctx
+    Execute Linux Command    echo "my secret" > secret.data
+    Execute Linux Tpm2 Tools Command    tpm2_sign -c key.ctx -o sig.rssa secret.data
+    Flush TPM Contexts
+    Execute Linux Tpm2 Tools Command    tpm2_verifysignature -c key.ctx -s sig.rssa -m secret.data
+    Execute Linux Command    rm -f key.pub key.priv key.ctx sig.rssa secret.data
+    Execute Reboot Command
+
+    ${setup_menu}=    Enter Setup Menu Tianocore And Return Construction
+    ${device_manager_menu}=    Enter Submenu From Snapshot And Return Construction
+    ...    ${setup_menu}
+    ...    Device Manager
+    Enter Submenu From Snapshot
+    ...    ${device_manager_menu}
+    ...    TCG2 Configuration
+    ${target_option_index}=    Search For The Option     ChangeEPS
+    Press Key N Times And Enter    ${target_option_index}    ${ARROW_DOWN}
+    Save Changes And Reset
+
```

Comment:
IIRC there will be a confirmation screen that asks to press some button, but you'll see it when launching the test on supported platform/TPM, it will be easier to add it then than to dig in the code to find out how it will look like.